scrapers/LegalPorno.yml

```diff
@@ -0,0 +1,31 @@
+name: "LegalPorno"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - legalporno.com/watch/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $description: //dl[@class="dl-horizontal scene-description__column"]
+    scene:
+      Title:
+        selector: //h1[@class="watchpage-title"]//text()
+        concat: " "
+      Date:
+        selector: //span[@title="Release date"]/a/text()
+        parseDate: 2006-01-02
+      Details: $description/div[3]/dd/text()
+      Performers:
+        Name: $description/div[1]/dd/a[position() mod 2 = 1]/text()
+      Studio:
+        Name: //div[@class="studio-director"]//a/text()
```

Comment:
same studio gets matched twice 
```
Name: //div[@class="col-md-4 col-lg-3 hide-mobile text-right"]/div[@class="studio-director"]//a/text()
```
limits that to once

Reply:
I had noticed that, but got lazy since it was always the same studio in my testing, and only the first match would get picked up anyway. Updated though